docs/pages/server-access/guides/openssh/openssh-manual-install.mdx

```diff
@@ -494,3 +494,19 @@ $ ssh -F ssh_config_teleport ${USER?}@node2.leafcluster.${CLUSTER}
   qualified domain name, rather than an IP address.
 
 </Admonition>
+
+### Upgrading to Teleport 14 and migrating from legacy OpenSSH nodes
```

Comment:
I don't think we need this note in this guide, as this guide does not apply to the legacy OpenSSH enrollment method

Reply:
What do you mean? The only difference between this and the original guide is if the registration process is largely automated or manual, I'm not sure why this shouldn't be included